src/ol_infrastructure/applications/concourse/__main__.py

```diff
@@ -41,19 +42,23 @@
     setup_vault_provider()
 concourse_config = Config("concourse")
 stack_info = parse_stack()
+env_name = f"{stack_info.env_prefix}-{stack_info.env_suffix}"
+
 network_stack = StackReference(f"infrastructure.aws.network.{stack_info.name}")
 policy_stack = StackReference("infrastructure.aws.policies")
 dns_stack = StackReference("infrastructure.aws.dns")
 consul_stack = StackReference(f"infrastructure.consul.operations.{stack_info.name}")
```

Comment:
This will need to be changed to be `f"infrastructure.consul.{stack_info.env_prefix}.{stack_info.name}"`